plugins/toolbox/src/components/Root/ToolsPage.tsx

```diff
@@ -217,11 +221,14 @@ export const ToolsPage = (props: ToolsPageProps) => {
                   style={!matchesSearch(tool) ? { display: 'none' } : {}}
                   wrapped
                   className={`${classes.fullWidth} ${classes.noPadding} ${classes.tab}`}
-                  label={tool.name}
+                  label={i18n_UNSAFE(`tool.${tool.id}.name`, tool.name)}
```

Comment:
https://www.i18next.com/translation-function/essentials#multiple-fallback-keys if it works here too